src/components/ContentSidebar/SidebarSection.scss

```diff
@@ -1,14 +1,21 @@
 @import '../variables';
 
-.bcs-section {
-    margin: 0 20px 20px 30px; // Right margin less as parent width leaves some gap for scrollbar
-}
-
 .bcs-section-content {
     padding: 15px 0 5px;
 }
 
+.bcs-section {
+    margin: 0 20px 20px 30px;
+    // background-color: $white;
+    // border: 1px solid $off-white;
+    // border-radius: 5px;
+    // box-shadow: 0 0 10px 0 $lightest-black;
+    // margin: 0 10px 15px 15px;
+    // padding: 10px;
```

Comment:
remove?

Reply:
Didn't remove these as they are the new design, so uncommenting these will get the new styles. Didnt want to forget what I did before.

Reply:
got it, maybe add some comments to explain why we want to keep them? someone else might accidentally remove these

Reply:
Actually I am just gonna nuke it for now, they are pretty easy to replicate